include/proxy-wasm/wasm.h

```diff
@@ -91,6 +94,12 @@ class WasmBase : public std::enable_shared_from_this<WasmBase> {
     return nullptr;
   }
 
+  // Capability restriction (restricting/exposing the ABI).
+  bool capabilityAllowed(std::string capability_name) {
+    return allowed_capabilities_.empty() ||
+           allowed_capabilities_.find(capability_name) != allowed_capabilities_.end();
```

Comment:
so basically if the configuration is null, all functions are allowed, and if it is the empty list, all functions are disabled, right? It seems a little bit confusing to me.

Reply:
But, as long as there would be no use cases for disabling all exported functions (and I don't come up with anything), this is just fine as it is.

Reply:
If it's empty then all functions are allowed. It does feel a little unintuitive, but yeah I don't think there's any case where you would want to have all capabilities restricted.